recipe-client-addon/jar.mn

```diff
@@ -3,8 +3,9 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 [features/shield-recipe-client@mozilla.org] chrome.jar:
-% resource shield-recipe-client %content/
-  content/lib/ (./lib/*)
-  content/data/ (./data/*)
-  content/skin/  (skin/*)
-  content/vendor/ (./vendor/*)
+% resource shield-recipe-client %
```

Comment:
Plus after that bug the public one will need `contentaccessible=yes` setting, but that's a separate issue...